benchmark.go

```diff
@@ -39,8 +40,36 @@ import (
 var (
 	errNegativeDuration = errors.New("duration cannot be negative")
 	errNegativeMaxReqs  = errors.New("max requests cannot be negative")
+
+	// using a global quantiles array mainly for ease of testing, and not passing the same array
+	// around to multiple functions
+	_quantiles = []string{"0.5000", "0.9000", "0.9500", "0.9900", "0.9990", "0.9995", "1.0000"}
```

Comment:
as chatted offline: usually we'd want to avoid a mutable global if possible: https://github.com/uber-go/guide/blob/master/style.md#avoid-mutable-globals. But I see that it'll simplify code much, and avoid duplicating the same array in test to print latencies for validation. so I am not strongly against this.